frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -239,13 +238,9 @@ export const onboardingLogic = kea<onboardingLogicType>([
             },
         ],
         shouldShowReverseProxyStep: [
-            (s) => [s.product, s.featureFlags, s.productKey],
-            (_product, featureFlags: FeatureFlagsSet, productKey) => {
-                const productsWithReverseProxy = []
-                if (featureFlags[FEATURE_FLAGS.REVERSE_PROXY_ONBOARDING] === 'test') {
-                    productsWithReverseProxy.push(ProductKey.FEATURE_FLAGS)
-                }
-                return productsWithReverseProxy.includes(productKey as ProductKey)
+            (s) => [s.productKey],
+            (productKey) => {
+                return productKey && [ProductKey.FEATURE_FLAGS].includes(productKey as ProductKey)
```

Comment:
is it easier to do  ` ProductKey.FEATURE_FLAGS === productKey`?

Reply:
This makes it easy to add another product.